pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -780,6 +782,15 @@ func (s *Service) reconcileELBTags(lb *infrav1.ClassicELB, desiredTags map[strin
 	return nil
 }
 
+func (s *Service) getClassicELBProtocol() *infrav1.ClassicELBProtocol {
+	elbProtocol := &infrav1.ClassicELBProtocolSSL
```

Comment:
We can default it to `SSL` in defaulting webhooks when it is not provided.

Reply:
Done